cinema/models.py

```diff
@@ -1,9 +1,33 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
+    related_name = "movies"
+    actors = models.ManyToManyField(
+        to=Actor,
+        related_name=related_name
```

Comment:
Choosing a clear and contextually meaningful related_name in Django, reflecting the purpose of the relationship, enhances code readability and maintainability

Reply:
> Choosing a clear and contextually meaningful related_name in Django, reflecting the purpose of the relationship, enhances code readability and maintainability

Do not really got it. You mean I should choose for actors related_name = "actors" and for genres "genres"?